runtest.py

```diff
@@ -33,6 +33,11 @@ def _notebook_run(path):
     return nb, errors
 
 def check_errors(expected, actual):
+    if len(expected) != len(actual):
+        print("len(expected) = " + str(len(expected)) + "\n")
```

Comment:
Woot

Reply:
lol is there a less dumb way to do this? python string interpolators are still new, right?

Reply:
Yeah I think f-interpolators are 3.6+ only :-(

The woot was mainly for the fact that tests now run :-)